api/src/paths/artifact/delete.ts

```diff
@@ -0,0 +1,125 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { SOURCE_SYSTEM } from '../../constants/database';
+import { SYSTEM_ROLE } from '../../constants/roles';
+import { getDBConnection, getServiceAccountDBConnection, IDBConnection } from '../../database/db';
+import { authorizeRequestHandler } from '../../request-handlers/security/authorization';
+import { ArtifactService } from '../../services/artifact-service';
+import { getKeycloakSource } from '../../utils/keycloak-utils';
+import { getLogger } from '../../utils/logger';
+
+const defaultLog = getLogger('paths/artifact/delete');
+
+export const POST: Operation = [
+  authorizeRequestHandler(() => {
+    return {
+      or: [
+        {
+          validSystemRoles: [SYSTEM_ROLE.SYSTEM_ADMIN, SYSTEM_ROLE.DATA_ADMINISTRATOR],
+          discriminator: 'SystemRole'
+        },
+        {
+          validServiceClientIDs: [SOURCE_SYSTEM['SIMS-SVC-4464']],
+          discriminator: 'ServiceClient'
+        }
+      ]
+    };
+  }),
+  deleteArtifact()
+];
+
+POST.apiDoc = {
+  description:
+    'Deletes artifacts. This will always return an object { success: boolean } to indicate if the deletion was successful or not.',
+  tags: ['artifact'],
+  security: [
+    {
+      Bearer: []
+    }
+  ],
+  requestBody: {
+    content: {
+      'application/json': {
+        schema: {
+          title: 'Artifacts to delete',
+          type: 'object',
+          required: ['artifactUUIDs'],
+          properties: {
+            artifactUUIDs: {
+              type: 'array',
+              items: {
+                type: 'string',
+                format: 'uuid'
+              },
+              minItems: 1
+            }
+          }
+        }
+      }
+    }
+  },
+  responses: {
+    200: {
+      description: '',
+      content: {
+        'application/json': {
+          schema: {
+            type: 'object',
+            required: ['success'],
+            properties: {
+              success: {
```

Comment:
Could just make the entire response be `true` or `false`, to be consistent with other endpoints that report success/failure in this way.